core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphQLActionBuilder.kt

```diff
@@ -732,7 +734,7 @@ object GraphQLActionBuilder {
             methodType.equals(GqlConst.QUERY_TYPE, true) -> GQMethodType.QUERY
             methodType.equals(GqlConst.MUTATION, true) -> GQMethodType.MUTATION
             else -> {
-                //TODO log warn
+                log.warn("Entry point is not found.")
```

Comment:
when issuing warning, be bit more descriptive

Reply:
done